filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -757,6 +779,15 @@ void OpenGLDriver::importTextureR(Handle<HwTexture> th, intptr_t id,
     CHECK_GL_ERROR(utils::slog.e)
 }
 
+void OpenGLDriver::updateRenderPrimitiveBuffer(GLRenderPrimitive* rp, GLVertexBuffer const* vb) {
+
+    // TODO: This will bind the VAO, then bind a new VBO to the appropriate slot. The implementation
+    // of this method will be similar to setRenderPrimitiveBuffer() except that it will not bother
+    // updating the index buffer.
+
+    rp->gl.vertexBufferVersion = vb->bufferObjectsVersion;
```

Comment:
maybe add a comment that this is called from draw() and must be as efficient as possible.